test-files/cases/attribute/unifying/highway-2927-bridge-1/Expected.osm

```diff
@@ -6544,12 +6589,11 @@
         <tag k="hoot:id" v="-25"/>
     </way>
     <way visible="true" id="-24" timestamp="1970-01-01T00:00:00Z" version="1">
-        <nd ref="-511"/>
-        <nd ref="-515"/>
-        <nd ref="-216"/>
-        <nd ref="-514"/>
         <nd ref="-513"/>
-        <nd ref="-26"/>
+        <nd ref="-514"/>
+        <nd ref="-216"/>
+        <nd ref="-515"/>
+        <nd ref="-511"/>
```

Comment:
Reordering and split off of node `-26` that was the good split above.

Reply:
Is this the footway in the bottom middle of the map that now is picking up "Av. Sur 4" for one section?  That part is better output as well.

Reply:
![screenshot from 2019-02-26 07-52-37](https://user-images.githubusercontent.com/303724/53413967-8ecca080-399b-11e9-8dca-8e9c7cd3b756.png)

This looks like another dogleg situation, similar to those in highway-2888.  The secondary geometry is much better than the ref there, unfortunately.